src/scripts/main.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
-// write code here
+const list = document.body.children[1];
+
+const employees = [];
+
+function salaryToNumber(salary) {
+  const slaryArray = salary.slice(1).split(',');
+  let result = '';
+
+  for (const element of slaryArray) {
+    result += element;
+  }
+
+  return +result;
+}
+
+function toFormat(data) {
+  let resultString = data.toString();
+  const result = [];
+
+  do {
+    if (resultString.length % 3 !== 0) {
+      result.push(resultString.slice(0, resultString.length % 3));
+      resultString = resultString.slice(resultString.length % 3);
+    }
+    result.push(resultString.slice(0, 3));
+    resultString = resultString.slice(3);
+  } while (resultString.length > 0);
+
+  return '$' + result.join(',');
+}
```

Comment:
No need to create this complicated function, it is redundant. You only need to sort DOM elements by the salary which was converted to a number.